### PR TITLE
[fix](planner)add IP type to compatibilityMatrix

### PR DESCRIPTION
**Pull request by starocean999**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by starocean999** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/catalog/Type.java [hunk 1]
```diff
@@ -1871,6 +1871,38 @@ public Integer getNumPrecRadix() {
         compatibilityMatrix[TIMEV2.ordinal()][IPV6.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[TIMEV2.ordinal()][AGG_STATE.ordinal()] = PrimitiveType.INVALID_TYPE;
 
+        // IPV4
+        compatibilityMatrix[IPV4.ordinal()][TIME.ordinal()] = PrimitiveType.INVALID_TYPE;
+        compatibilityMatrix[IPV4.ordinal()][TIMEV2.ordinal()] = PrimitiveType.INVALID_TYPE;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/catalog/Type.java:1876]: add some cases?

**Comment by starocean999** [comment]
run buildall

**Comment by starocean999** [comment]
run buildall

**Review by wm1581066 — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**